wic/src/components/FoodLocations.js

```diff
@@ -1,48 +1,42 @@
 import React, {PropTypes, Component} from 'react';
 import ReactDOM from 'react-dom';
-
-import shouldPureComponentUpdate from 'react';
-
-import Gmap from './Gmap';
-
+import ResoruceMap from './ResoruceMap';
 import ResourceTable from './ResourceTable';
 
-const markers = [{
-    position: {
-      lat: 25.0112183,
-      lng: 121.52067570000001,
-    },
-    key: `Taiwan`,
-    defaultAnimation: 2,
-}];
-
 class FoodLocations extends React.Component {
     constructor(props) {
         super(props);
-        this.state = {
-            apiResultData: ''
-        };
+        this.state = {};
     }
 
-    componentWillReceiveProps() {
-      console.log('componentWillReceiveProps');
-      var self = this;
-      fetch('https://api.smc-connect.org/search?category=Food Pantry&location=' + this.props.zipcode).then(function (response) {
+    findFoodLocations() {
+      const self = this;
+      const url = `https://api.smc-connect.org/search?category=Food Pantry&location=${this.props.zipcode}`;
+      return Promise.resolve(true).then(() => {
```

Comment:
I think we're transpiling, so, async and await might be something we could use.  You should check them out:
https://www.twilio.com/blog/2015/10/asyncawait-the-hero-javascript-deserved.html
https://ponyfoo.com/articles/understanding-javascript-async-await